CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 7.85
 -----
+*   New Features
+    *   Add suggested folders support.
```

Comment:
Hey @MiSikora Shouldn't we call this as Smart Folders instead of Suggested Folders?